envpool/sokoban/sokoban_envpool.cc

```diff
@@ -75,13 +75,8 @@ void SokobanEnv::Step(const Action& action_dict) {
   current_step_++;
 
   const int action = action_dict["action"_];
-  if (action == kActNoop) {
-    WriteState(static_cast<float>(reward_step_));
-    return;
-  }
-  // From here on, assume the agent will try to move
 
-  const int change_coordinates_idx = (action - 1) % kChangeCoordinates.size();
+  const int change_coordinates_idx = (action) % kChangeCoordinates.size();
```

Comment:
```suggestion
  const int change_coordinates_idx = action % kChangeCoordinates.size();
```

Reply:
Action should always be less than 4. If it's not, the `.at()` below should error and that's what we want.

Reply:
yeah, I wanted to make that change but thought it was okay to leave it.

I'll make that change now!